src/components/accessories/admission/admissionForm/AdmissionForm.tsx

```diff
@@ -57,7 +57,9 @@ const AdmissionForm: FC<AdmissionProps> = ({
   const admissionTypes = useSelector(
     (state: IState) => state.admissionTypes.allAdmissionTypes.data
   );
-  const wards = useSelector((state: IState) => state.wards.allWards.data);
+  const wards = useSelector((state: IState) =>
+    state.wards.allWards.data?.filter((ward) => !(ward.opd ?? false))
```

Comment:
as the issues states, the wards to be filtered are the ones with 0 `beds`, not the ones with `opd=true` because that it only tell us if the ward is accepting OPD visits.